jekyll/_cci2/getting-started.md

```diff
@@ -107,7 +107,7 @@ Reference](https://circleci.com/docs/2.0/configuration-reference).
 
 Let's get a bit more complex. Let's edit our `.circleci/config.yml` file now. On
 GitHub, it is possible to edit files directly. Use the URL below and substitute
-the name of your repository and username (replace the text with`{backets}`) and
+the name of your repository and username (replace the text within `{brackets}`) and
```

Comment:
```suggestion
the name of your repository and username (replace the text with `{brackets}`) and
```

This should actually be `with` as we need the user to replace the brackets as well, not just the content within.